webview/src/plots/components/util.ts

```diff
@@ -47,76 +44,3 @@ export const getNbItemsPerRow = (size: PlotSize) => {
 
 export const shouldUseVirtualizedGrid = (nbItems: number, size: PlotSize) =>
   nbItems > MaxItemsBeforeVirtualization[size]
-
-const truncateTitlePart = (title: string, size: number) =>
```

Comment:
[F] Moved this to `extension/src/plots/vega/util.ts` and took the tests from `App.test.tsx` and put them into the associated test file.